staging/src/k8s.io/component-base/logs/klogflags/klogflags.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 202 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2022 The Kubernetes Authors.
```

Reply:
Oh, the good old times of 202! You are right of course, good catch.